swap/src/database.rs

```diff
@@ -126,9 +127,12 @@ pub async fn open_db(
                 sqlite.insert_peer_id(swap_id, peer_id).await?;
             }
 
+            tracing::info!("Sucessfully migrated data to sqlite! Using sqlite.");
```

Comment:
Nice! Should we add a line that prints the path to the sled DB so users know what to delete?

Reply:
yep thats a good idea